app/views/users/invitations/otp_visible.js.erb

```diff
@@ -0,0 +1,9 @@
+$("#otp").css({ visibility: 'visible', display: 'block' }) 
```

Comment:
@ajaystha42 

this file and app/views/users/registrations/otp_visible.js.erb are the same. Create 1 partial and reuse it